### ISSUE TITLE
Add / update warning message for Iframe component

### ISSUE THREAD

**Issue by fernandolucchesi** [equinor/energyvision#1113]
With the introduction of CSP Headers, it is needed to whitelist all dependencies that our websites use. Consequently, if an editor creates an iframe to a non-whitelisted source, the iframe will not work.

We should add a warning message for the Editors to make editors aware of this.

In a not so distant future we should also create a process for editors to request whitelisting of urls (we could potentially have this set in Sanity too)

**Comment by wenche** [comment]
LGTM :thumbsup:

Let's see if this makes it easier to remember these things.

**Comment by padms** [comment]
@wenche  Yes, I agree that it warns for all the iframes. But then I thought it should warn anyway, instead of not warning at all.

**Comment by BirteThornquist** [comment]
@padms @wenche I think it is sufficient to have the information text for the Frame URL field, but maybe shorten it a bit and make it more "to the point". What about: "Any URL must be whitelisted to load. Please make sure to verify that the iframe loads correctly before publishing, otherwise contact dev team for whitelisting."

**Comment by padms** [comment]
done

**Comment by wenche** [comment]
@meols  Padmaja has added a descriptive text for the iframe URL field that hopefully will remind you of the whitelist issue for iframes and/or explain why an iframe is not working as expected when you preview. :)

![Skjermbilde 2022-05-30 kl. 15.21.37.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/e5972e27-7b79-430a-aaf0-2fe56637dfcc)
